starter-code/css/main.css

```diff
@@ -7,4 +7,99 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-color: #1c5380;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #1c5380;
+}
+
+#brand {
+  width: 50%;
+  height: 50%;
+}
+
+#home {
+  background-color: #1c5380;
+  color: white;
+  border-style: none;
+  color: white;
+  box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+}
+
+#phone {
+  background-image: url("../images/phones.png");
+  width: 274px;
+  background-size: cover;
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+}
+/* 
+div {
+  border: 2px solid black;
+} */
+section {
+  display: flex;
+  justify-content: center;
+  padding-top: 90px;
+}
+
+button {
+  border-style: none;
+  padding: 0;
+  background-color: white;
+  margin: 10px 10px;
+}
+
+.content {
+  width: 450px;
+  height: 50%;
+  padding-left: 70px;
+}
+
+h1 {
+  color: #06365f;
+}
+
+p {
+  color: #06365f;
+  font-size: 15px;
+  font-weight: 500;
+}
+
+li {
+  list-style: none;
+  color: #1c5380;
+  text-transform: uppercase;
+  font-weight: 600;
+  font-size: 10px;
+}
+
+li a {
+  text-decoration: none;
+  color: #1c5380;
+  text-transform: uppercase;
+}
+
+ul {
+  display: flex;
+  justify-content: space-around;
+}
+
+footer {
+  margin: 5% 30%;
+}
+
+.text {
+  box-shadow: 0px 0 4px 0px grey;
+  padding: 1px 25px;
+  background-color: white;
+}
+
+.white-box {
+  background-color: white;
```

Comment:
This technique worked here because we have a fixed screen size, but it might get tricky with bigger or changing website heights.
A more scalable way would have been to create a div with a colored background, placed at the very top of your screen. Its CSS could have been something like:

```css
.background-div {
  position: absolute;
  top: 0; /*along with position: absolute, places your div at 0px from the top border*/
  z-index: -1; /*tells your div to stay in the backgorund*/
  width: 100%;
  height: 250px;
  background-image: url( '../images/bgtile.png' ); /*sets your div's background image to the bgtile, which will create the nice gradient over repetition*/
}
```